pkg/translator/azure/property_names.go

```diff
@@ -0,0 +1,78 @@
+package azure
+
+var mappings = map[string]map[string]string{
+	"common": {},
+	"FrontDoorAccessLog": {
+		"trackingReference":   "",
```

Comment:
nit: you might consider mapping it to `az.service_request_id` - we use it in the Azure SDKs to record unique id returned by the service in response - https://github.com/Azure/azure-sdk/blob/main/docs/tracing/distributed-tracing-conventions.md#http-client-spans